### PR TITLE
Post/mjml deeper look

### LINKED ISSUE #20: Additional content for the MJML post
While setting mjml up for the MyWings page I came across the case where we are using a general email layout (in `app/views/layouts/mailer.html.erb`), which is included then in all the emails. 

Since this is the standard setup in a rails app, I thought that adding [the solution](https://github.com/sighmon/mjml-rails/issues/4) about how to do this would complete Hugo's blog post. 

@HugoGiraudel: what do you think?

### PR DESCRIPTION
**Pull request by zfhui**
Wrote about my recent learnings of MJML while integrating it to MyWings. 

ping @edenspiekermann/devs 

closes #20

### CODE REVIEW

File: _posts/2017-02-24-diving-deeper-into-mjml.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+{% highlight erb %}
+<!--  ./app/views/user_mailer/welcome_email.mjml.erb -->
+<mj-section>
+  <mj-column>
+    <mj-text> Hello <%= @name %></mj-text>
+  </mj-column>
+</mj-section>
+{% endhighlight %}
+
+Pay attention to the changes of the file formats (when and where to use `.mjml` or `.mjml.erb`), it otherwise won't work!
```
Comment (nburkley) [inline _posts/2017-02-24-diving-deeper-into-mjml.md]: `it otherwise won't work` =>  `otherwise it won't work`

Comment (nburkley) [inline _posts/2017-02-24-diving-deeper-into-mjml.md]: Maybe `Pay attention to the different file formats`?

File: _posts/2017-02-24-diving-deeper-into-mjml.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+          -->
+          Hello World! 
+        </mj-text>
+      </mj-section>
+    </mj-container>
+  </mj-body>
+</mjml>
+{% endhighlight %}
+
+Another way to apply styling is to use `mj-style` and write plain css in it. You can then use `mj-raw` to write plain html and use your css class:
```
Comment (nburkley) [inline _posts/2017-02-24-diving-deeper-into-mjml.md]: should you `css.upcase`?

File: _posts/2017-02-24-diving-deeper-into-mjml.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+        </mj-column>
+      </mj-section>
+    </mj-container>
+  </mj-body>
+</mjml>
+{% endhighlight %}
+
+But in one of our projects, we are loading fonts which are nested in the Rails assets folder structure `./app/assets/fonts/*` and we used to load it with `@font-face` into our email templates. 
+
+To allow this again we had to hack MJML a bit, since we weren't able to find anything else to allow this.
```
Comment (nburkley) [inline _posts/2017-02-24-diving-deeper-into-mjml.md]: Might be better with a `,` after again. `To allow this, we again had to...`

File: _posts/2017-02-24-diving-deeper-into-mjml.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+    </mj-container>
+  </mj-body>
+</mjml>
+{% endhighlight %}
+
+## Wrapping Up
+
+We are still very happy with MJML and are trying to integrate it into more and more of our projects. 
+
+The version 3 upgrade helps a lot in terms of DRY up the code and to integrate standard styling elements. But when the design becomes more complex, we did reach some of the limits of MJML. But the [lively discussions and issue requests online](https://github.com/mjmlio/mjml) promise that new features are soon to come.
```
Comment (nburkley) [inline _posts/2017-02-24-diving-deeper-into-mjml.md]: would it be better to link to the issues page?

**Comment by zfhui** [comment]
@nburkley Cool, that was quick! I applied your suggestions. Thanks!

**Pull request merged**